NodeJS/src/AgentSkill.ts

```diff
@@ -0,0 +1,47 @@
+const args = require('minimist')(process.argv.slice(2));
```

Comment:
what is going on?

Reply:
This will help us to parse the run time parameters. 
https://www.npmjs.com/package/minimist